jetty-deploy/src/main/config/modules/global-webapp-common.d/global-webapp-common.xml

```diff
@@ -7,12 +7,10 @@
       <Arg>
         <New class="org.eclipse.jetty.deploy.bindings.GlobalWebappConfigBinding">
           <Set name="jettyXml">
-            <Property>
-              <Name>jetty.deploy.webappCommonConfig</Name>
-              <Default>
-                <Property name="jetty.base"/>/etc/webapp-common.xml
-              </Default>
-            </Property>
+            <Call name="resolvePath" class="org.eclipse.jetty.xml.XmlConfiguration">
+              <Arg><Property name="jetty.base"/></Arg>
+              <Arg><Property name="jetty.deploy.webappCommonConfig"/>/etc/webapp-common.xml</Arg>
```

Comment:
I don't think this is correct (was wrong before too). The `*.mod` file property under `[ini-template]` already has `etc/webapp-common.xml`, so this should be:

```xml
<Arg><Property name="jetty.deploy.webappCommonConfig" default="etc/webapp-common.xml" /></Arg>
```

Reply:
That wouldn't work for people that use jetty home without a care about the pwd (like what is seen with tooling such as apache daemon)...

Eg:

```
$ java -Djetty.home=$JETTY_HOME -Djetty.base=$JETTY_BASE -jar $JETTY_HOME/start.jar
```

That is a valid usage, and the raw `etc/webapp-common.xml` would resolve against whatever the `System.getProperty("user.dir")` value is.